sycl/include/CL/sycl/detail/pi.hpp

```diff
@@ -347,6 +348,7 @@ class DeviceBinaryImage {
   DeviceBinaryImage::PropertyRange SpecConstIDMap;
   DeviceBinaryImage::PropertyRange DeviceLibReqMask;
   DeviceBinaryImage::PropertyRange KernelParamOptInfo;
+  DeviceBinaryImage::PropertyRange AssertUsed;
```

Comment:
IIRC adding a member here is ABI breaking change. Could you please change it to something like getSpecConstantsDefaultValues?

Reply:
I think this should have been catched with some test.
Anyway, I've removed this field.